docs/meshstack.meshmarketplace.metering.md

```diff
@@ -130,31 +130,6 @@ The line items for the seller will appear in the usage reports and chargeback st
 
 This feature can be used for services for which usage should be tracked but the chargeback process is not yet completely established.
 
-<!--snippet:mesh.kraken.meshMarketplace-->
```

Comment:
Was this diff intended to be in this PR?

Reply:
no, i just wasn't on the latest state of develop and therefore snippet creation failed and i just removed it here. I reverted this change